Scripts/Python/xTakableClothing.py

```diff
@@ -358,6 +359,7 @@ def OnNotify(self,state,id,events):
                 PtDebugPrint("xTakableClothing: Guild set to:", guildSDLValues[base])
             if not self.IItemInCloset():
                 avatar.avatar.addWardrobeClothingItem(base,ptColor().white(),ptColor().white())
+                PtSendKIMessage(kKILocalChatStatusMsg,PtGetLocalizedString("ACA.KI.NewClothing"))
```

Comment:
The translation should probably be in the KI loc files, not the ACA one. Also, I think it would be really neat if he message told you what exactly was added to the closet. You could do that by changing the loc text to `%1s was added to your wardrobe." and changing this line to:

```suggestion
                PtSendKIMessage(kKILocalChatStatusMsg, PtGetLocalizedString("ACA.KI.NewClothing", [self.IGetItem(base).description]))
```

Then you would see something like:
`DRC Vest was added to your wardrobe.`

Reply:
Figured the ACA as it was a clothing item, but I can put it in KI instead.

Reply:
It's not a bad thought, I just seem to remember that there are a few KI messages like this one already in the KI file :smile:

Reply:
Okay, changes made.